### ISSUE TITLE
Log Exeception after application started

### ISSUE THREAD

**Issue by joinusdevops-account** [slydeveloper/spring-boot-admin-docker#2]
Hello,

I noticed some strange java exception in logs, it started to log, after 1-2 min from start. It's occurred also in the default configuration:

```
 :: Spring Boot ::       (v2.1.10.RELEASE)

2020-04-16 06:55:16.557  INFO 1 --- [           main] p.s.t.s.SpringBootAdminDockerApplication : Starting SpringBootAdminDockerApplication v1.1 on 91f41752af9a with PID 1 (/opt/spring-boot-admin-docker/app.jar started by root in /opt/spring-boot-admin-docker)
2020-04-16 06:55:16.561  INFO 1 --- [           main] p.s.t.s.SpringBootAdminDockerApplication : The following profiles are active: docker
2020-04-16 06:55:19.811  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 8080 (http)
2020-04-16 06:55:19.861  INFO 1 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2020-04-16 06:55:19.862  INFO 1 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.27]
2020-04-16 06:55:20.075  INFO 1 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2020-04-16 06:55:20.075  INFO 1 --- [           main] o.s.web.context.ContextLoader            : Root WebApplicationContext: initialization completed in 3365 ms
2020-04-16 06:55:21.140  INFO 1 --- [           main] o.s.s.concurrent.ThreadPoolTaskExecutor  : Initializing ExecutorService 'applicationTaskExecutor'
2020-04-16 06:55:21.373  WARN 1 --- [           main] ion$DefaultTemplateResolverConfiguration : Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2020-04-16 06:55:22.471  INFO 1 --- [           main] o.s.s.web.DefaultSecurityFilterChain     : Creating filter chain: any request, [org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter@4e31276e, org.springframework.security.web.context.SecurityContextPersistenceFilter@57576994, org.springframework.security.web.header.HeaderWriterFilter@2a225dd7, org.springframework.security.web.authentication.logout.LogoutFilter@5c669da8, org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter@4b7dc788, org.springframework.security.web.authentication.www.BasicAuthenticationFilter@2812b107, org.springframework.security.web.savedrequest.RequestCacheAwareFilter@67b9b51a, org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter@490caf5f, org.springframework.security.web.authentication.AnonymousAuthenticationFilter@1a72a540, org.springframework.security.web.session.SessionManagementFilter@125290e5, org.springframework.security.web.access.ExceptionTranslationFilter@7c51f34b, org.springframework.security.web.access.intercept.FilterSecurityInterceptor@6d026701]
2020-04-16 06:55:22.551  INFO 1 --- [           main] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 2 endpoint(s) beneath base path ''
2020-04-16 06:55:22.678  INFO 1 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
2020-04-16 06:55:22.685  INFO 1 --- [           main] p.s.t.s.SpringBootAdminDockerApplication : Started SpringBootAdminDockerApplication in 7.169 seconds (JVM running for 8.148)
2020-04-16 06:55:25.508  INFO 1 --- [nio-8080-exec-1] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring DispatcherServlet 'dispatcherServlet'
2020-04-16 06:55:25.508  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Initializing Servlet 'dispatcherServlet'
2020-04-16 06:55:25.516  INFO 1 --- [nio-8080-exec-1] o.s.web.servlet.DispatcherServlet        : Completed initialization in 8 ms



^@2020-04-16 06:56:51.505 ERROR 1 --- [nio-8080-exec-5] o.a.catalina.connector.CoyoteAdapter     : Exception while processing an asynchronous request

java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:513) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.Request.action(Request.java:430) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:242) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:861) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1579) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_111-internal]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_111-internal]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_111-internal]

^@2020-04-16 06:57:03.909 ERROR 1 --- [nio-8080-exec-8] o.a.catalina.connector.CoyoteAdapter     : Exception while processing an asynchronous request

java.lang.IllegalStateException: Calling [asyncError()] is not valid for a request with Async state [MUST_DISPATCH]
	at org.apache.coyote.AsyncStateMachine.asyncError(AsyncStateMachine.java:440) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessor.action(AbstractProcessor.java:513) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.Request.action(Request.java:430) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.catalina.core.AsyncContextImpl.setErrorState(AsyncContextImpl.java:401) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.catalina.connector.CoyoteAdapter.asyncDispatch(CoyoteAdapter.java:239) ~[tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessor.dispatch(AbstractProcessor.java:242) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:53) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:861) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1579) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) [na:1.8.0_111-internal]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) [na:1.8.0_111-internal]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61) [tomcat-embed-core-9.0.27.jar!/:9.0.27]
	at java.lang.Thread.run(Thread.java:745) [na:1.8.0_111-internal]

```
My properties file:

```
#General setup

spring.boot.admin.monitor.default-retries=2

spring.boot.admin.monitor.status-interval=120000ms
spring.boot.admin.monitor.status-lifetime=120000ms

spring.boot.admin.monitor.info-interval=2m
spring.boot.admin.monitor.info-lifetime=2m

# Spring Boot server port
server.port=8080

# Spring Boot Admin user/uassword
spring.security.user.name=admin
spring.security.user.password=secret

# Spring Boot Admin title
spring.boot.admin.ui.title=Admin

# custom property for disable security
spring.boot.admin.security.enabled=true

spring.boot.admin.discovery.enabled=true
spring.boot.admin.discovery.converter.management-context-path=/manage

spring.boot.admin.routes.endpoints=env,metrics,dump,info,configprops,trace,logfile,refresh,loggers,auditevents
spring.boot.admin.ui.brand=<img src="assets/img/icon-spring-boot-admin.svg"><span>Cloud Admin  AU AWS</span>

admin-user=admin
admin-password=password
actuator-user=actuator-user
actuator-password=actuator-Password

# ENDPOINTS (AbstractEndpoint subclasses)
endpoints.actuator.enabled=true
endpoints.actuator.sensitive=true
management.security.enabled=true
endpoints.health.sensitive=true
endpoints.info.sensitive=true
management.context-path=/manage

# slack notification to #uat channel
#spring.boot.admin.notify.slack.webhook-url=


#spring.cloud.discovery.client.simple.instances.cloud-config-server[0].uri=http://appserver:8080/appname
#spring.cloud.discovery.client.simple.instances.cloud-config-server[0].metadata.user.name=actuator-user
#spring.cloud.discovery.client.simple.instances.cloud-config-server[0].metadata.user.password=actuator-Password
```

**Comment by slydeveloper** [comment]
It looks that issue is related with Tomcat and Spring Boot https://github.com/codecentric/spring-boot-admin/issues/1039
Will try to fix it

**Comment by slydeveloper** [comment]
Released new Docker image version: 1.2
Based on:
Spring Boot Admin version: 2.2.2
Spring Boot version: 2.2.6.RELEASE
